x-pack/plugins/infra/public/pages/metrics/index.tsx

```diff
@@ -15,7 +15,7 @@ import { IIndexPattern } from 'src/plugins/data/common';
 import { MetricsSourceConfigurationProperties } from '../../../common/metrics_sources';
 import { DocumentTitle } from '../../components/document_title';
 import { HelpCenterContent } from '../../components/help_center_content';
-import { Header } from '../../components/header';
+import { useReadOnlyBadge } from '../../hooks/use_readOnly_badge';
```

Comment:
@weltenwort Yep good catch. I actually caught this one and already fixed locally while I started looking into the existing functional tests. It looks like existing functional tests for badge are failing, nothing related to my changes though. 

For example this test fails https://github.com/elastic/kibana/blob/master/x-pack/test/functional/apps/infra/feature_controls/infrastructure_security.ts#L176, cause the badge is missing on Home page. Badge appears fine on Observability pages. Same is true for badge related tests of other apps as well (I tested uptime and it also fails). I am really wondering if these tests are known to be failing or we just spotted a bug on the home page? I will look into that more anyway in case I broke something that affected the global header.

Also this test could be improved I guess. It should first navigate into the Observability page and then check for the existence of the badge on that page and not home page. I'll look into how I can change this. 

<img width="1253" alt="Screenshot 2021-06-30 at 00 14 51" src="https://user-images.githubusercontent.com/2852703/123875390-df70b180-d939-11eb-87d4-842a35c2e80a.png">

Reply:
@weltenwort Yep I plan to remove `Header` component. I think it's being used in a few more places.